libs/ui-watt/src/lib/foundations/typography/+storybook/storybook-typography-overview.component.ts

```diff
@@ -14,121 +14,140 @@
+    weight: 'Semi-bold',
+    letterCase: 'Sentence',
     letterSpacing: '0',
-    lineHeight: '48px*',
   },
   {
-    name: 'Headline 3',
     html: typographyHtmlSnippets.h3.tag,
-    size: '28px',
-    weight: 'Semibold (600)',
-    letterCase: 'All caps',
+    size: 'M',
+    weight: 'Semi-bold',
+    letterCase: 'Sentence',
     letterSpacing: '0',
-    lineHeight: '42px*',
   },
   {
-    name: 'Headline 4',
     html: typographyHtmlSnippets.h4.tag,
-    size: '24px',
-    weight: 'Semibold (600)',
-    letterCase: 'All caps',
+    size: 'M',
+    weight: 'Semi-bold',
+    letterCase: 'Sentence',
     letterSpacing: '0',
-    lineHeight: '36px*',
   },
   {
-    name: 'Headline 5',
     html: typographyHtmlSnippets.h5.tag,
-    size: '20px',
-    weight: 'Semibold (600)',
-    letterCase: 'All caps',
+    size: 'M',
+    weight: 'Semi-bold',
+    letterCase: 'Sentence',
     letterSpacing: '0',
-    lineHeight: '30px*',
   },
   {
-    name: 'Lead (text-l)',
     html: typographyHtmlSnippets.textL.class,
-    size: '18px',
-    weight: 'Semibold (600)',
+    size: 'L',
+    weight: 'Regular',
     letterCase: 'Sentence',
     letterSpacing: '0',
-    lineHeight: '27px*',
   },
   {
-    name: 'Body (text-m)',
     html: typographyHtmlSnippets.bodyTextM.tag,
-    size: '16px',
-    // eslint-disable-next-line sonarjs/no-duplicate-string
-    weight: 'Regular (400)',
+    size: 'M',
+    weight: 'Regular',
     letterCase: 'Sentence',
     letterSpacing: '0',
-    lineHeight: '24px*',
   },
   {
-    name: 'Small (text-s)',
     html: typographyHtmlSnippets.textS.tag,
-    size: '14px',
-    weight: 'Regular (400)',
+    size: 'S',
+    weight: 'Regular',
     letterCase: 'Sentence',
     letterSpacing: '0',
-    lineHeight: '21px*',
   },
   {
-    name: 'Extra small (text-xs)',
-    html: typographyHtmlSnippets.textXs.class,
-    size: '12px',
-    weight: 'Regular (400)',
+    html: typographyHtmlSnippets.textLHighlighted.class,
+    size: 'L',
+    weight: 'Bold',
+    letterCase: 'Sentence',
+    letterSpacing: '0',
+  },
+  {
+    html: typographyHtmlSnippets.bodyTextMHighlighted.tag,
+    size: 'M',
+    weight: 'Bold',
+    letterCase: 'Sentence',
+    letterSpacing: '0',
+  },
+  {
+    html: typographyHtmlSnippets.textSHighlighted.tag,
+    size: 'S',
+    weight: 'Bold',
     letterCase: 'Sentence',
     letterSpacing: '0',
-    lineHeight: '18px*',
   },
   {
-    name: 'Button',
     html: typographyHtmlSnippets.button.storybook,
-    size: '14px',
-    weight: 'Semibold (600)',
-    letterCase: 'All caps',
-    letterSpacing: '1.25px',
-    lineHeight: '16px',
+    size: 'M',
+    weight: 'Bold',
+    letterCase: 'Sentence',
+    letterSpacing: '0',
   },
   {
-    name: 'Label',
     html: typographyHtmlSnippets.label.class,
-    size: '12px',
-    weight: 'Semibold (600)',
+    size: 'S',
+    weight: 'Bold',
     letterCase: 'All caps',
-    letterSpacing: '1.5px',
-    lineHeight: '14.4px',
+    letterSpacing: '1.25px',
```

Comment:
suggestion:nitpick: "button" and "label" on small screen have "bold" `font-weight` in Figma (though they still appear to be "semi-bold"). Align with Kenneth what should be the correct `font-weight`.

![image](https://user-images.githubusercontent.com/1096332/153609208-e6c42a83-8683-4a64-843c-5170599036d5.png)